### PR TITLE
Restructured Convars file

### PR DESCRIPTION
**Pull request by JMorell**
Not tested ingame.

### CODE REVIEW

**Review by xen-000 — CHANGES_REQUESTED** [review]

**Review by Vauff — COMMENTED** [review]
I think we agreed on hungarian notation? Since Lua doesn't have strict type checking.

**Comment by JMorell** [comment]
I need to make another pull request?

**Review by xen-000 — APPROVED** [review]
Yeah that'll do.

File: scripts/vscripts/ZombieReborn/Convars.lua [hunk 1]
```diff
@@ -1,17 +1,35 @@
...
-iKnockbackScale = 5
-Convars:RegisterConvar("zr_knockback_scale", tostring(iKnockbackScale), "Knockback damage multiplier", 0)
\ No newline at end of file
+Convars:RegisterConvar("zr_infect_spawn_type", tostring(Config.Infect.SpawnType), "Type of Mother Zombies Spawn [0 = MZ spawn where they stand, 1 = MZ get teleported back to spawn on being picked]", 0)
+Convars:RegisterConvar("zr_infect_spawn_time_min", tostring(Config.Infect.SpawnTimeMin), "Minimum time in which Mother Zombies should be picked, after round start", 0)
+Convars:RegisterConvar("zr_infect_spawn_time_max", tostring(Config.Infect.SpawnTimeMax), "Maximum time in which Mother Zombies should be picked, after round start", 0)
+Convars:RegisterConvar("zr_infect_spawn_mz_ratio", tostring(Config.Infect.SpawnMZRatio), "Ratio of all Players to Mother Zombies to be spawned at round start", 0)
+Convars:RegisterConvar("zr_infect_spawn_mz_min_count", tostring(Config.Infect.SpawnMZMinCount), "Minimum amount of Mother Zombies to be spawned at round start", 0)
+
+Convars:RegisterConvar("zr_knockback_scale", tostring(knockbackConfig.Scale), "Knockback damage multiplier", 0)
```
Comment (Vauff) [inline scripts/vscripts/ZombieReborn/Convars.lua:22]: This should be `Config.Knockback.Scale`

**Review by Vauff — CHANGES_REQUESTED** [review]

File: scripts/vscripts/ZombieReborn/Convars.lua (hunk 1 above)
Comment (JMorell) [reply scripts/vscripts/ZombieReborn/Convars.lua]: fixed

File: scripts/vscripts/ZombieReborn/Infect.lua [hunk 2]
```diff
@@ -75,9 +77,10 @@ function Infect_PickMotherZombies()
     print("Player count: " .. iPlayerCount .. ", Mother Zombies Spawned: " .. iMotherZombieCount)
 end
 
-function Infect_OnRoundFreezeEnd()
-    local iMZSpawntimeMinimum = (Convars:GetInt("zr_infect_spawn_time_min") or iInfectSpawntimeMin)
-    local iMZSpawntimeMaximum = (Convars:GetInt("zr_infect_spawn_time_max") or iInfectSpawntimeMax)
+-- hook this to OnRoundStart in main script
+function Infect_OnRoundStart()
```
Comment (Vauff) [inline scripts/vscripts/ZombieReborn/Infect.lua:81]: These 2 lines got reverted when you solved the merge conflict. See https://github.com/Source2ZE/ZombieReborn/commit/f3899667b3ef54c8d074a7dd987ec95659a3df08

**Review by Vauff — CHANGES_REQUESTED** [review]

**Review by Vauff — APPROVED** [review]

**Pull request merged**